pgxn/neon/neon.c

```diff
@@ -614,11 +614,15 @@ _PG_init(void)
 	ExecutorEnd_hook = neon_ExecutorEnd;
 }
 
+/* Various functions exposed at SQL level */
+
 PG_FUNCTION_INFO_V1(pg_cluster_size);
 PG_FUNCTION_INFO_V1(backpressure_lsns);
 PG_FUNCTION_INFO_V1(backpressure_throttling_time);
 PG_FUNCTION_INFO_V1(approximate_working_set_size_seconds);
 PG_FUNCTION_INFO_V1(approximate_working_set_size);
+PG_FUNCTION_INFO_V1(neon_get_lfc_stats);
+PG_FUNCTION_INFO_V1(local_cache_pages);
```

Comment:
Would be nice to consolidate on naming. We use `lfc` in one place and `local_cache` in another.

Reply:
I agree. Unfortunately these names are exposed at the SQL level, so we're stuck with them :-(. Changing would require modifying the .sql installation scripts, bumping the version, and possibly breaking any monitoring scripts or other stuff we have that calls these.